### PR TITLE
CL node keys are not displayed in the node ops UI when a key exists that isn't part of a configured chain

### PR DESCRIPTION
**Pull request by hans-kessock**
https://smartcontract-it.atlassian.net/browse/BCF-1991

Evaluated many different ways to fix/modify this - ultimately went with the simplest route, and for once, kicking the can down the road is actually the right thing to do.

This is a fix for node side configuration state that is NOT considered an error, but the operator-ui is built to assume would be an error. Eth Keys in storage that are not part of a currently configured chain are not considered orphaned although they may seem that way.

### CODE REVIEW

File: core/web/resolver/query.go [hunk 1]
```diff
@@ -409,20 +409,26 @@ func (r *Resolver) ETHKeys(ctx context.Context) (*ETHKeysPayloadResolver, error)
...
+			It is not an error to have ETH keys persisted which are not part of a currently
+			configured chain; ergo, this should not cause listing the keys that have currently
+			configured chains to fail
+			TBD: Revisit when we have a comprehensive strategy for what can be 'orphaned keys'
+		*/
+		if err == nil {
+			ethKeys = append(ethKeys, ETHKey{
+				addr:  k.EIP55Address,
+				state: state,
+				chain: chain,
```
Comment (jmank88) [inline core/web/resolver/query.go:423]: Can/should we include the keys still, but without `chain` set? b/c we do this already for the special case of `evm.ErrNoChains` above ^

Comment (hans-kessock) [reply core/web/resolver/query.go:423]: It doesn't match up to the operator-ui key management section these keys fall under - titled "EVM Chain Accounts", and it seemed that the 'no chains' condition was a legacy error condition that is no longer possible error - but, importantly, I'm not 100% certain - so I didn't remove it.

**Comment by hans-kessock** [comment]
Small addition to fix mismatch of enums between resolver chain types and node chain types

**Comment by hans-kessock** [comment]
There's a fix in the operator-ui branch that is associated with these fixes that will merge after this PR.

**Review by george-dorin — DISMISSED** [review]

**Pull request merged**